lib/models/ObjectMD.js

```diff
@@ -883,6 +883,24 @@ class ObjectMD {
         return this;
     }
 
+    /**
+     * Set object legal hold status
+     * @param {boolean} legalHold - true if legal hold is 'ON' false if 'OFF'
+     * @return {ObjectMD} itself
+     */
+    setLegalHold(legalHold) {
```

Comment:
I know its a bit late but javascript supports getters and setters now.

Reply:
That's right, `set legalhol(val)` `get legalHold()`works as well; I thought it might worth to keep it consistent across the ObjectMD so just went with the flow.